build.sbt

```diff
@@ -1,18 +1,16 @@
-lazy val scalaTestVersion = "3.2.9"
+lazy val scalaTestVersion = "3.2.10"
+
+ThisBuild / crossScalaVersions := Seq("2.11.12", "2.12.15", "2.13.6", "3.0.2")
+ThisBuild / scalaVersion := crossScalaVersions.value.head
```

Comment:
this might a change? I think currently implicitly the last version is used, so `sbt 'show scalaVersion'` goes to 3.0.0 now. I don't mind it goes to 2.11.x now, just noting the difference.

Reply:
I think you're right. I'll change it before merging.